uftrace.h

```diff
@@ -424,6 +424,8 @@ enum uftrace_msg_type {
  */
 enum uftrace_agent_msg {
 	UAM_CLOSE = (1U << 0), /* Close the connection with the client */
+
+	UAM_TRACE = (1U << 5), /* Tracing state (on/off) */
```

Comment:
Please just use the next integer rather than make a hole.

Reply:
I'm doing it. Just to clarify, and wanted to separate internal messages - such as `close` or later `data size` - from real options like `trace=on|off`. I wanted to fit the internal messages in later commits in the hole, rather than reorder options each time I need to add one.

Reply:
I'm not sure if it's meaningful to separate the internal and public messages.  Hmm.. how about making a message can handle many options?  Something like below?

```c
enum uftrace_agent_msg {
    UAM_CLOSE, /* close the connection */
    UAM_QUERY, /* query supported features/options */
    UAM_GET_OPT, /* get current option value */
    UAM_SET_OPT, /* set new option value */
    ...
};
```

Each message should handle variable-length data then.